frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashItem.tsx

```diff
@@ -96,51 +58,55 @@ export const NewInUnleashItem = ({
-            <NewInUnleashTooltip
-                open={open}
-                onClose={handleTooltipClose}
-                title={label}
-                longDescription={longDescription}
-                onCheckItOut={onCheckItOut}
-                docsLink={docsLink}
-                preview={preview}
-                beta={beta}
-            >
-                <StyledItemButton>
-                    {icon}
-                    <LabelWithSummary>
-                        <StyledItemTitle>
-                            <Typography fontWeight='bold' fontSize='small'>
-                                <Truncator title={label} arrow>
-                                    {label}
-                                </Truncator>
-                            </Typography>
-                            <ConditionallyRender
-                                condition={beta}
-                                show={<Badge color='secondary'>Beta</Badge>}
-                            />
-                        </StyledItemTitle>
-                        <Typography fontSize='small'>{summary}</Typography>
-                    </LabelWithSummary>
-                    <Tooltip title='Dismiss' arrow sx={{ marginLeft: 'auto' }}>
-                        <StyledItemButtonClose
-                            aria-label='dismiss'
-                            onClick={onDismissClick}
-                            size='small'
-                        >
-                            <Close fontSize='inherit' />
-                        </StyledItemButtonClose>
-                    </Tooltip>
-                </StyledItemButton>
-            </NewInUnleashTooltip>
+        <ListItem disablePadding>
+            {popout ? (
+                <>
+                    <NewInUnleashDialog
+                        open={open}
+                        onClose={handleTooltipClose}
+                        title={label}
+                        longDescription={longDescription}
+                        onCheckItOut={onCheckItOut}
+                        docsLink={docsLink}
+                        preview={preview}
+                        beta={beta}
+                    ></NewInUnleashDialog>
+                    <NewInUnleashSideBarItem
+                        label={label}
+                        summary={summary}
+                        icon={icon}
+                        beta={beta}
+                        onClick={onOpen}
+                        onDismiss={onDismissClick}
+                    />
+                </>
+            ) : (
+                <NewInUnleashTooltip
+                    open={open}
+                    onClose={handleTooltipClose}
+                    title={label}
+                    longDescription={longDescription}
+                    onCheckItOut={onCheckItOut}
+                    docsLink={docsLink}
+                    preview={preview}
+                    beta={beta}
+                >
+                    <NewInUnleashSideBarItem
```

Comment:
Tooltips do actually require the children to be mounted inside them. I have a lot of opinions about this, as a backend developerl web browsers were a mistake